server/lease/metrics.go

```diff
@@ -49,11 +49,58 @@ var (
 			// 1 second -> 3 months
 			Buckets: prometheus.ExponentialBuckets(1, 2, 24),
 		})
+
+	leaseAttached = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "etcd_debugging",
+		Subsystem: "lease",
+		Name:      "attach_total",
+		Help:      "The number of leases that are attached to a lease item.",
+	})
+
+	leaseDetached = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "etcd_debugging",
+		Subsystem: "lease",
+		Name:      "detach_total",
+		Help:      "The number of leases that are detached from a lease item.",
+	})
+
+	initLeaseCount = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
Why we need this metric?

Reply:
Replied [here](https://github.com/etcd-io/etcd/pull/18711/files/ba35d471dfae1de5fd002069c4d055afb230f142#r1853090002) but we could also roll this up into another metric but I also don't want to affect the behavior of existing metrics, hence created a separate metric.